### ISSUE TITLE
[Trinket] [154172]Aman'Thul's Vision

### ISSUE THREAD

**Issue by Ulthalesh** [holydeew/WoWHellgarve-Legion#176]
Hello I am doing this report because the Aman'Thul's Vision trinket has a display problem, the passive that only works in antorus  the burning throne who is called Aman'Thul's greatness in the character's buffs it gives strength while as on the screen I am a healer  it should display intelligence however it gives the exact number in main statistic according to the specialization if we display the character panel.

Screen: 
https://imgur.com/fy1MPhK
https://imgur.com/06qx6Z3

**Comment by Gueryghembal** [comment]
Yes ! as a monk i also have this issue, i take "some" screens to see if it was pure unluck or not.

**Date :** June 08, 2023

**Screens :**

> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/18e42ffc-8d44-46a5-b8bf-3e495413aff7)

Here above the normal way of the trinket.

Here the screen from a monk character :

> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/9fb7ddac-43a2-4c7b-8dfd-b68240d30ccd)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/2d00c740-4fbd-4693-b1ac-802d8a404515)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/af0a58e9-253d-4e3b-b46a-04302bf0d752)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/7d89d864-d802-4df8-8d1c-3e36af214cdc)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/6c3ca626-cca9-40be-b2a4-dedc68fdf294)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/04635876-d626-4817-8d41-27be3a25b4b9)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/dc85ea36-93aa-4dde-81ff-6a894a1b786d)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/407be17c-97b9-4bd9-b42a-09dff63e33f0)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/c7f82976-53a1-4ff0-851d-b1fbcd77bc47)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/25c970b6-6734-49ee-8715-c0017665b521)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/4048a681-08cc-4aea-9662-61496a01d1cf)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/dac2edf4-7eba-4761-9c5c-5e2cae5cac66)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/c1be5f75-7171-4288-ab54-3ad4ccafdf52)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/0bda0d7a-f729-43bc-a153-6bb41c98fa77)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/6bec633e-ce50-42aa-91b7-dfb9e9db6f48)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/26b57a44-d81a-4c3c-98e0-13570eb17247)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/c1b077a9-ed04-4c8c-b3c5-1325558afb10)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/0063a5bf-9f21-4b0e-8a32-4b8c0882f1a1)
> ![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/558d8c30-e3d2-4e4a-9ed7-862024f8daaf)


As a monk i should have agility and not strength ! (Force = Strength in french)


/!\ IN FACT THE TRINKET IS WORKING ! It's just the visual /!\

Here the proofs :

![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/af0a18e1-154b-40cf-a567-d1dd5224c6dd)

![image](https://github.com/holydeew/WoWHellgarve-Legion/assets/105439669/55ccaa0e-ffad-431d-a9d5-f8dddf14dd7d)

So it's good ! But can fool peoples.

**Comment by skeezerbean** [comment]
So this trinket works fine, but just displays the wrong thing in the buff tooltip?
